<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWidget</class>
 <widget class="QWidget" name="LoginWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>290</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Log in</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../Resources.qrc">:/image/Resource/logo.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLineEdit" name="addressEdit">
     <property name="text">
      <string>127.0.0.1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>Server</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QSpinBox" name="portEdit">
     <property name="maximum">
      <number>65635</number>
     </property>
     <property name="value">
      <number>18189</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QLineEdit" name="loginEdit">
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>Login</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QLineEdit" name="passwordEdit">
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="placeholderText">
      <string>Password</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="statusLabel">
     <property name="text">
      <string>Status</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>243</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="connectButton">
     <property name="text">
      <string>Log in</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Resources.qrc"/>
 </resources>
 <connections/>
</ui>
